handler/skill_test.go

```diff
@@ -151,7 +157,40 @@ func TestPerformGetError(t *testing.T) {
 	w := httptest.NewRecorder()
 	err := performGet(w, r)
 	if err == nil {
-		t.Errorf("Expecting error for TestPerform")
+		t.Errorf("Expecting error for TestPerformGetError")
+	}
+}
+
+func TestGetSkillsFiltered(t *testing.T) {
+	skillsConnector = data.NewAccessor(MockDataAccessor{})
+	r := httptest.NewRequest(http.MethodGet, "/skills?skilltype=scripted", nil)
+
+	w := httptest.NewRecorder()
+	err := performGet(w, r)
+	if(err != nil) {
+		fmt.Errorf("Did not expect error when getting skills with filter")
```

Comment:
These should all be t.Errorf() then testing.  fmt.Errorf() creates a new error object with formatted text, but doesn't return it to the testing tool.